contrib/container-fedora-25/custodia.conf

```diff
@@ -0,0 +1,42 @@
+[DEFAULT]
```

Comment:
the config file example that we already have, is not enough for this case? https://github.com/latchset/custodia/blob/master/contrib/config/custodia/custodia.conf

Or should we just update it, instead of add a new one?